### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by denigit5**
- I have added the footer section to my portfolio project.  
- The footer includes the form and the submit button.
- Please review and let me know if I require changes.

### CODE REVIEW

**Review by didierganthier — COMMENTED** [review]
Hi @denigit5 and @MPDADDY ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project that doesn't have a professional README. The README needs to include all the main sections of the `Table of Contents`, 📖  yours is missing the `Key Features` section for now. ❌ 

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @denigit5 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 
You have submitted a project, that has no commit history from your pair programming partners, 

This milestone is a [pair-programming activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#work-and-submission-mode). Kindly ensure that everyone in your pair-programming group contributes to the project you're reviewing, 

We will check the commit history to make sure that everybody has contributed code on each project. 🤗🛠

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by didierganthier** [comment]
Hi @abdulhamiid are you sure it has no commit history? Because I see two different people committing. Please double check.

File: index.html [hunk 1]
```diff
@@ -143,7 +143,34 @@ <h4 class="connect2">LET'S CONNECT</h4>
                 <img class="arrow2" src="images/files/DisabledArrow.png">
                 <div class="divider"></div> <br> <br> <br> <br> <br> <br>
               </li>
-        </ul>   
+        </ul> 
+        <footer class="footer">
+            <section class="footer-container">
+                <h2 class="footer-heading">Contact me</h2>
+                <p class="footer-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it!</p>
+                <form action="https://formspree.io/f/{form_id}" method="post">
```
Comment (Owusu-Desmond) [inline index.html:151]: - [ ] Please make sure that your form is submitted successfully, Currently when I submit your form, the form can't be found as you can see in the screenshot below. I this you don't have the `form_id` in your form URL `https://formspree.io/f/{form_id}` which is generated by [formspree.io](https://formspree.io). Kindly fix this :pray: 

  Screenshot from 2023-05-10 09-56-20

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @denigit5 and @MPDADDY , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -143,7 +143,34 @@ <h4 class="connect2">LET'S CONNECT</h4>
...
+                            <input id="name"  maxlength="30" placeholder="Lorem ipsum" type="text" value="Lorem ipsum" required>
+                        </label>
+                    </li>
+                    <li>
+                        <label for="email">
+                            <input id="email"  type="email" placeholder="loremipsum@gmail.com" value="loremipsum@gmail.com" required>
+                        </label>
+                    </li>
+                    <li>
+                        <textarea name="comments" id="comments" cols="30" rows="10" placeholder="write your message here..." required></textarea>
```
Comment (LionRouge1) [inline index.html:164]: [OPTIONAL]
- [ ] Kindly add `maxlength="500"` attribute to the message field to make sure that the message will not passe 500 characters as requested in the project requirement.

_This is a requirement of the project but first reviewer didn't notice it. so I requested this as an optional but I highly recommend you to make the change_

**Review by LionRouge1 — APPROVED** [review]
Hi @denigit5, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**